### PR TITLE
Replaces a sleep with a timer in late joining

### PR DESCRIPTION
**Pull request by lzimann**

### CODE REVIEW

**Comment by Robustin** [comment]
I was gonna say remove it because its redundant but whatever works.

**Comment by Robustin** [comment]
Uh I went ahead and did your change and I'm getting:

Runtime in timer.dm,234: Cannot modify /mob/dead/new_player/proc/AddEmploymentContract.active_timers. proc name: New (/datum/timedevent/New)

**Comment by lzimann** [comment]
I'll test it when I get home later

File: code/modules/mob/dead/new_player/new_player.dm [hunk 1]
```diff
@@ -363,7 +363,7 @@
 			SSshuttle.arrivals.QueueAnnounce(humanc, rank)
 		else
 			AnnounceArrival(humanc, rank)
-		AddEmploymentContract(humanc)
+		addtimer(CALLBACK(.proc/AddEmploymentContract, humanc), 30)
```
Comment (Cyberboss) [inline code/modules/mob/dead/new_player/new_player.dm]: addtimer(CALLBACK(src, .proc/AddEmploymentContract, humanc), 30)

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/mob/dead/new_player/new_player.dm (hunk 1 above)
Comment (lzimann) [inline code/modules/mob/dead/new_player/new_player.dm]: Oh right, forgot about the first arg lol

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Line... Beginnings???

**Pull request closed without merging**